### ISSUE TITLE
mozjpeg removes EXIF/ITPC data from PNG files

### ISSUE THREAD

**Issue by robrwo** [mozilla/mozjpeg#375]
I've run the mozjpeg build from the master branch, with PNG support enabled. It appears to remove IPTC data from PNG files.

**Comment by kornelski** [comment]
Yeah, the demo `cjpeg` command just copies the pixels out of input images. It's generally terrible all-around, so I recommend compiling ImageMagick with MozJPEG support, or using something more sophisticated like https://github.com/imazen/imageflow
